Arcs/Arcs.cpp

```diff
@@ -74,32 +94,32 @@ PROGRAM " " PACKAGE_VERSION "\n"
 "       --dist_tsv=FILE     write min/max distance estimates to FILE\n"
 "       --samples_tsv=FILE  write intra-contig distance/barcode samples to FILE\n";
 
-static const char shortopts[] = "f:a:B:s:c:Dl:z:b:g:m:d:e:r:v";
+
+
+static const char shortopts[] = "f:a:B:s:c:Dl:z:b:g:m:d:e:r:vt:u:j:k:";
 
 enum {
     OPT_HELP = 1,
     OPT_VERSION,
-    OPT_BX,
```

Comment:
I know that the `--bx` argument isn't required anymore but I'd like to keep it in for backward-compatibility. I just don't want the code to fail if a user specifies that option.

Reply:
Ok adding it again.